app/views/sample_types/_form.html.erb

```diff
@@ -51,13 +50,11 @@
                                                                            display_isa_tag: false } %>
           <% end %>
 
-          <% unless @sample_type.uploaded_template? || !@sample_type.editing_constraints.allow_new_attribute? %>
```

Comment:
check for @sample_type.uploaded_template? should remain for now, but might be revisited as part of #1766